requirements.txt

```diff
@@ -1 +1,5 @@
-requests
\ No newline at end of file
+certifi==2019.11.28
+chardet==3.0.4
+idna==2.9
+requests==2.23.0
+urllib3==1.25.8
```

Comment:
Is this version of urllib necessary? `urllib` is already one of Python 3's standard libraries.